azure/template.json

```diff
@@ -0,0 +1,566 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "resourceEnvironmentName": {
+            "type": "string"
+        },
+        "environmentName": {
+            "type": "string"
+        },
+        "sharedFrontendAppServicePlanName": {
+            "type": "string"
+        },
+        "sharedBackEndAppServicePlanName": {
+            "type": "string"
+        },
+        "sharedResourceGroupName": {
+            "type": "string"
+        },
+        "aspSize": {
+            "type": "string",
+            "defaultValue": "1"
+        },
+        "aspInstances": {
+            "type": "int",
+            "defaultValue": 1
+        },
+        "aspSku": {
+            "type": "string",
+            "defaultValue": "Basic"
+        },
+        "configurationStorageConnectionString": {
+            "type": "securestring"
+        },
+        "loggingRedisConnectionString": {
+            "type": "securestring"
+        },
+        "serviceName": {
+            "type": "string"
+        },
+        "apiCustomHostname": {
+            "type": "string"
+        },
+        "apiCertificateName": {
+            "type": "string"
+        },
+        "sharedKeyVaultName": {
+            "type": "string"
+        },
+        "appServiceAllowedIPs": {
+            "type": "array",
+            "defaultValue": [
+            ]
+        },
+        "apiStorageConnectionString": {
+            "type": "securestring"
+        },
+        "logLevel": {
+            "type": "string"
+        },
+        "loggingRedisKey": {
+            "type": "securestring"
+        },
+        "idaAudience": {
+            "type": "string"
+        },
+        "idaTenant": {
+            "type": "string"
+        },
+        "uiCustomHostname": {
+            "type": "string"
+        },
+        "uiCertificateName": {
+            "type": "string"
+        },
+        "redisConnection": {
+            "type": "securestring"
+        },
+        "idamsRealm": {
+            "type": "string"
+        },
+        "idamsADFSMetadata": {
+            "type": "string"
+        },
+        "redisSessionProviderConnectionString": {
```

Comment:
```suggestion
        "defaultSessionRedisConnectionString": {
```

Reply:
It's using this package: https://www.nuget.org/packages/SFA.DAS.Web.RedisSessionStateProvider/ which uses a provider with name RedisSessionProvider so would prefer to keep the name to match appSettings